### PR TITLE
Added support for actionData

### PR DESCRIPTION
**Pull request by andrewha12**

### CODE REVIEW

**Review by plaird — APPROVED** [review]
Thanks for the contribution! Verified it is back compat with Bazel 3.x. No NPEs if actionData is missing.

**Comment by plaird** [comment]
I will copybara this to the bazel-eclipse repo tonight-ish.

File: sdk/bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/bep/event/BEPBuildMetricsEvent.java [hunk 1]
```diff
@@ -94,7 +113,7 @@ void parseDetails(JSONObject metricsDetail) {
     public String toString() {
         return "BEPBuildMetricsEvent [actionsCreated=" + actionsCreated + ", actionsExecuted=" + actionsExecuted
                 + ", usedHeapSizePostBuild=" + usedHeapSizePostBuild + ", cpuTimeInMs=" + cpuTimeInMs
-                + ", wallTimeInMs=" + wallTimeInMs + ", analysisPhaseTimeInMs=" + analysisPhaseTimeInMs + ", index="
+                + ", wallTimeInMs=" + wallTimeInMs + ", analysisPhaseTimeInMs=" + analysisPhaseTimeInMs + ", actionData=" + actionData + ", index="
```
Comment (andrewha12) [inline sdk/bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/bep/event/BEPBuildMetricsEvent.java:116]: Currently this does not print each property of each action in the order as shown in the format above due iterating through the keys of the map not being in order of insertion. Not sure if I should make the print follow the order shown above?

Comment (plaird) [reply sdk/bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/bep/event/BEPBuildMetricsEvent.java:116]: No, print order of attributes is irrelevant for toString(). The toString() is intended just for logging/pretty print on pages.

**Pull request merged**